integer.js

```diff
@@ -0,0 +1,27 @@
+const prompt = require('prompt-sync')();
+
+const x = parseInt(prompt('please assign an integer to x: '));
+
+let value_of_x, value_of_y;
+
+if (x > 0) {
+  value_of_x = true
+} else {
+  value_of_x = false
+}
+
+const y = parseInt(prompt('now please assign an integer to y: '));
+
+if (y > 0) {
+  value_of_y = true
+} else {
+  value_of_y = false
+}
+
+if ((value_of_x == true) && (value_of_y == true)) {
+  console.log("two variables are > 0");
+} else if (((value_of_x == true) && (value_of_y == false)) || ((value_of_x == false) && (value_of_y == true))) {
```

Comment:
Here, instead of comparing the two cases with each other, and their possibilities, we could potentially shorten the code, by removing both cases, and instead comparing 2 conditions in one case. 

```suggestion
else if((value_of_x == true) || (value_of_y == true)){
```
This fix right here, produces the same result as the else if statement you wrote down. The only difference is the length of the code, which allows for the simplest solution. The reason this code works is due to the fact how javascript interprets it. Since the condition is checked from left to right, if the first statement is false, the whole condition is false. If the first statement is true, the whole condition is true, and hence this code works the same as yours!